pkg/controller/applicationmonitoring/applicationmonitoring_controller.go

```diff
@@ -429,6 +451,10 @@ func (r *ReconcileApplicationMonitoring) createOrUpdateAdditionalScrapeConfig(cr
 		"integreatly.yaml": []byte(job),
 	}
 
+	flatList := common.Flatten()
+	cr.Status.Bbthash = fmt.Sprintf("%x", md5.Sum([]byte(joinQuote(flatList))))
+	r.client.Update(context.TODO(), cr)
```

Comment:
Missing error check

Reply:
Also this is probably not necessary as the CR Phase is updated in a later step which updates the CR anyway

Reply:
Hmm I don't think the CR gets updated in a later step here, this is amongst the last few steps in the reconcile loop for this phase, there is another update section which is updating/creating a ``scrapeConfigSecret`` is that what you mean ?

Reply:
Right, it's only the scrape config secret that get's updated.